libpam/pam.pc.in

```diff
@@ -0,0 +1,9 @@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: PAM
+Description: The primary library of the linux-PAM project. Used for the creation of PAM modules and applications.
```

Comment:
`The primary Linux-PAM library. It is used by PAM modules and PAM-aware applications.`

Reply:
It does sound better.